src/extlibs/hrvo/visualizer/hrvo_visualizer.py

```diff
@@ -0,0 +1,194 @@
+import argparse
+import math
+import os
+
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib.patches import Circle
+from matplotlib.animation import FuncAnimation, PillowWriter
+
+# Constants
+FRAME_RATE = 30
+FRAME_LENGTH_MS = 1000.0 / FRAME_RATE
+PLAY_BACK_SPEED = 1.0
+
+PLOT_SCALE_CONSTANT = 0.125
+
+DEFAULT_ROBOT_COLOR = "royalblue"
+COLLIDED_ROBOT_COLOR = "red"
+
+FILE_DIRECTORY = "/tmp"
```

Comment:
I think this should be passed in as an arg

Reply:
Updated the file directory to be a user passed argument